### PR TITLE
Update FPGA ANR ref design readme

### PR DESCRIPTION
**Pull request by jkinsky**
# Existing Sample Changes
## Description

Restructured to match the new template. Updated the markdown to conform to standard support. Moved several sections around, split Windows and Linux build and run information into separate sections, and added detail to the DevCloud section.

Fixes Issue# 

## External Dependencies

N/A

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [x] Documentation change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Command Line
- [x] oneapi-cli
- [x] VSCode

### CODE REVIEW

**Comment by jkinsky** [comment]
Adding @IgorOchocki and @jimmytwei for notifications.

File: DirectProgramming/DPC++FPGA/ReferenceDesigns/anr/README.md [hunk 1]
```diff
@@ -1,257 +1,286 @@
-# Adaptive Noise Reduction (ANR)
-This SYCL*-compliant reference design demonstrates a highly optimized image sensor adaptive noise reduction (ANR) algorithm for field programmable gate arrays (FPGAs).
+# `Adaptive Noise Reduction` Sample
+This `Adaptive Noise Reduction` (ANR) is a reference design to demonstrate a highly optimized image sensor adaptive noise reduction algorithm for field programmable gate arrays (FPGAs).
```
Comment (yuguen-intel) [inline DirectProgramming/DPC++FPGA/ReferenceDesigns/anr/README.md:2]:
```suggestion
This `Adaptive Noise Reduction` (ANR) sample is a reference design to demonstrate a highly optimized image sensor adaptive noise reduction algorithm for field programmable gate arrays (FPGAs).
```

File: DirectProgramming/DPC++FPGA/ReferenceDesigns/anr/README.md [hunk 2]
```diff
@@ -1,257 +1,286 @@
...
+  |Available Nodes    |Command Options
+  |:---               |:---
+  |FPGA Compile Time  |`qsub -l nodes=1:fpga_compile:ppn=2 -d .`
+  |FPGA Runtime       |`qsub -l nodes=1:fpga_runtime:ppn=2 -d .`
```
Comment (yuguen-intel) [inline DirectProgramming/DPC++FPGA/ReferenceDesigns/anr/README.md:254]: One suggestion given by one of the FAEs I talked to regarding these readme changes would be to have the FPGA runtime line split in two with the commands to get an Arria 10 FPGA and a Stratix 10 FPGA:
```suggestion
  |FPGA Runtime (Arria 10)       |`qsub -l nodes=1:fpga_runtime:arria10:ppn=2 -d .`
  |FPGA Runtime (Stratix 10)       |`qsub -l nodes=1:fpga_runtime:stratix10:ppn=2 -d .`
```

**Review by yuguen-intel — APPROVED** [review]
Looks good to me 👍

File: DirectProgramming/DPC++FPGA/ReferenceDesigns/anr/README.md (hunk 1 above)
Comment (jkinsky) [reply DirectProgramming/DPC++FPGA/ReferenceDesigns/anr/README.md:2]: Good catch. Thanks.

**Review by ishaghosh27 — APPROVED** [review]

**Pull request merged**